XcodeServerSDK/XcodeServerConfig.swift

```diff
@@ -33,16 +42,51 @@ public class XcodeServerConfig : JSONSerializable {
         return dict
     }
     
-    public init(var host: String, user: String?, password: String?) {
+    ///
+    /// Initializes a server configuration with the provided host.
```

Comment:
Not true; I've always favored java `/***/` over `///` but while documenting a personal framework and also writing this [blog post](www.estebantorr.es/blog/2015/06/14/swift-2-dot-0-throws-tip/) I came around some issues that javadoc was not correctly parsing the documentation whereas `///` was working as expected; hence why I now use `///` to document.